internal/cli/input/secret.go

```diff
@@ -95,17 +97,23 @@ func GetAmazonCredentials(allowSessionToken bool) (string, map[string]string, er
 		return "", nil, err
 	}
 
-	if !allowSessionToken && value.SessionToken != "" {
-		return "", nil, errors.New("AWS session tokens are not supported by Banzai Cloud Pipeline")
-	}
-
-	out := map[string]string{
-		"AWS_ACCESS_KEY_ID":     value.AccessKeyID,
-		"AWS_SECRET_ACCESS_KEY": value.SecretAccessKey,
-	}
+	var out map[string]string
 
-	if allowSessionToken {
-		out["AWS_SESSION_TOKEN"] = value.SessionToken
+	if assumeRole != "" {
+		value, err := stscreds.NewCredentials(session, assumeRole).Get()
+		if err != nil {
+			return "", nil, err
+		}
+		out = map[string]string{
+			"AWS_ACCESS_KEY_ID":     value.AccessKeyID,
+			"AWS_SECRET_ACCESS_KEY": value.SecretAccessKey,
+			"AWS_SESSION_TOKEN":     value.SessionToken,
+		}
+	} else {
+		out = map[string]string{
```

Comment:
I'd keep the ability to use a session token without assuming a role. (Use case: user runs the cli on a shared jump host, and neither wants to use their original iam credentials, nor to assume a role.)

Reply:
Good point!